extension/models/Logs/mage.js

```diff
@@ -0,0 +1,51 @@
+const util = require('util')
+
+const MAGENTO_ERROR_TITLE = 'Got error from magento'
+
+/**
+ * @param {Logger} log
+ * @param {Response} response
+ * @param {Object} responseBody
+ */
+function error (log, response, responseBody) {
```

Comment:
interestingly enough, you want to pass the context here and use context.log.error instead, see architecture board: https://wiki.shopgate.guru/display/AB/Coding+standards